pkg/client/core/service_parameter.go

```diff
@@ -16,6 +21,33 @@ type parameterService struct {
 	report  client.ParameterReport
 }
 
+func (s *parameterService) DeleteSecret(ctx context.Context, provider v1alpha1.CloudProvider, name string) error {
+	err := s.spinner.Start("parameter")
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err = s.spinner.Stop()
+	}()
+
+	fmt.Println(name)
```

Comment:
Intentional printline - considering sending (o.Err down here) or something else, even though this should be refactored as soon as possible.